lib/assessment_redux_10_28.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+// Question 1
+// Write a function that takes zero or more arguments
+// and returns the minimum from that set
+// assume that all the argument are numbers
+var min = function() {
+  var result = arguments[0];
+  for (i = 1; i < arguments.length; i++){
```

Comment:
`i = 1` here is an assignment to an undeclared variable. The interpreter will look for `i` in outer scopes, until it reaches the global scope, and if it doesn't find it there, it will create it. If it does find it somewhere, it will **overwrite it** with 1, interfering with code elsewhere in your program. This is solved by replacing the assignment statement `i = 1` with the variable declaration and initialization `var i = 1`.